common/src/cams/test-utilities/auxiliary.ts

```diff
@@ -0,0 +1 @@
+export const urlRegex = /https?:\/\/.*\//;
```

Comment:
This is a very small file. I know I created this but does this belong somewhere else? or are there other things that possbily belong here?

Reply:
I can't say I'm a fan of a filed named `auxiliary`, so I'm not opposed to moving it.

Reply:
The question is ... where should we move it to?

Reply:
We have that very constant also defined in the UI `test-utilities.ts`. I'm all for consolidation. Maybe outside the test-utilities files since they are `jest` and `vitest` specific.

Reply:
moved to regex.ts